web/src/components/molecules/index.tsx

```diff
@@ -1,2 +1,4 @@
 export { EntityTableHeader, EntityTableHeaderProps } from './entity';
```

Comment:
already specified in the lower folder, could be ```
export * from './core';
export * from './entity';
```

Reply:
👍 👍 👍 .

Reply:
61f74a738784c26382c8f0c2358f4dc60f48bc63